devops/openshift/forms-deploy.yml

```diff
@@ -170,7 +170,19 @@ objects:
         name: ${NAME}
       minReplicas: "${{REPLICAS}}"
       maxReplicas: 6
-      targetCPUUtilizationPercentage: 50
+      metrics:
+          - type: Resource
+            resource:
+              name: cpu
+              target:
+                type: AverageValue
+                averageValue: '0.8'
+          - type: Resource
+            resource:
+              name: memory
+              target:
+                type: AverageValue
+                averageValue: "1Gi"
```

Comment:
I see higher numbers for form.io server than API for autoscaling, is this with consideration that form.io server is going to have lighter load than API?

Reply:
No, the averageValue is a percentage of the CPU and memory limits and here for formio, we have used the 50% as we used in the previous load testing iterations, this inturn helps for pods scaling without many requests failing.

Reply:
Got it. Thanks for explaining. If 50% then cpu must be 0.75 ?

Reply:
Yeah you are right, i did increased a bit and tested around so gave those value.